BrainPortal/app/controllers/groups_controller.rb

```diff
@@ -151,8 +151,11 @@ def update #:nodoc:
       end
     end
 
+    unless (current_user.available_users.map{ |u| u.id } | @group.user_ids).include?(params[:group][:creator_id].to_i )
```

Comment:
Add logging of creator_id change in controller (this can be done simply by providing the attribute name to `update_attributes_with_logging` below